docs/pages/versions/unversioned/sdk/sqlite.mdx

```diff
@@ -474,7 +474,7 @@ The `expo-sqlite` library is designed to be a solid SQLite foundation. It enable
 
 [Drizzle](https://orm.drizzle.team/) is a ["headless TypeScript ORM with a head"](https://orm.drizzle.team/docs/overview). It runs on Node.js, Bun, Deno, and React Native. It also has a CLI companion called [`drizzle-kit`](https://orm.drizzle.team/kit-docs/overview) for generating SQL migrations.
 
-Check out the [Drizzle ORM documentation](https://orm.drizzle.team/) and the [`expo-sqlite` integration guide](https://orm.drizzle.team/docs/get-started-sqlite#expo-sqlite) for more details.
+Check out the [Drizzle ORM documentation](https://orm.drizzle.team/) and the [`expo-sqlite` integration guide](https://orm.drizzle.team/docs/connect-expo-sqlite) for more details.
```

Comment:
Hello @adamrhunter, thank you for catching this and the update! 👍  

Wdyt about linking to getting started guide for Expo SQLite instead of connection guide?

```suggestion
Check out the [Drizzle ORM documentation](https://orm.drizzle.team/) and the [`expo-sqlite` integration guide](https://orm.drizzle.team/docs/get-started/expo-new) for more details.
```

Reply:
@Simek  - Agreed - the getting started guide is a better landing point.